docs/ocis/adr/0023-index-and-store-metadata.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "23. Index and store metadata"
+date: 2023-10-17T15:15:00+01:00
+weight: 23
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/adr
+geekdocFilePath: 0023-index-and-store-metadata.md
+---
+
+
+* Status: proposed
+* Deciders: @butonic, @micbar
+* Date: 2023-10-17
+
+## Context and Problem Statement
+
+ownCloud Infinite Scale is supposed to become a data platform and as such it needs to provide access to metadata.
+Currently only metadata available for all file types (filesize, mime-type, ...) is stored in the index and the metadata storage.
+We want to make other metadata available to consumers of our internal and external APIs. Simple examples would be audio metadata
+like artist, album and title or exif metadata in images.
```

Comment:
This I find misleading. In our discussion you said that you were focusing on a set of metadata for known file types (audio...) that have a representation in the standard Graph API as of today.

This does not talk about a "generic" interface for meta data. I think that would be important to mention.

Reply:
fair point, I'll try to improve

Reply:
I've read it a few times again and I don't think it's misleading. That we focus on the graph api is not part of the problem statement, but part of the decision imho... what do you think?

FWIW I've slightly adjusted the wording here and added a note to the outcome.